grid/social-media/posts-lookup/profile.supr

```diff
@@ -231,7 +184,7 @@ model Post {
   text string
   """
   Reply ID
-  If the post is reply it contains ID of parent post
+  If the post is reply it contains an ID of the parent post
```

Comment:
... **a** reply

Reply:
```suggestion
  ID of a parent post in case the post is a reply.
```

Actually this is quite confusing property name since it's _parent's_ ID, not an ID of the reply @janhalama.
Renaming this would be breaking change but we can duplicate it and remove it with major version.